sormas-e2e-tests/src/test/resources/features/sanity/web/ContactFilters.feature

```diff
@@ -115,4 +115,108 @@ Feature: Contact filter functionality
+    And I open the first found result in the CHOOSE SOURCE window for DE version
+    Then I click SAVE button on Edit Contact Page
+    Then I click on the Contacts button from navbar
+    And I apply Id of last created Contact on Contact Directory Page
+    And I apply Contact classification filter to "Bestätigter Kontakt" on Contact Directory Page
+    And I apply Disease of source filter "COVID-19" on Contact Directory Page
+    And I apply Disease variant filter to "Sequenzierung ausstehend" on Contact Directory Page
+    And I apply Classification of source case filter to "0. Nicht klassifiziert" on Contact Directory Page
+    And I apply Follow-up status filter to "Nachverfolgung abgebrochen" on Contact Directory Page
+    And I click APPLY BUTTON in Contact Directory Page
+    And I check that number of displayed contact results is 1
+    And I apply Id of last created Contact on Contact Directory Page
+    And I apply Contact classification filter to "Kein Kontakt" on Contact Directory Page
+    And I click APPLY BUTTON in Contact Directory Page
+    And I check that number of displayed contact results is 0
+    And I apply Contact classification filter to "Bestätigter Kontakt" on Contact Directory Page
+    And I apply Disease of source filter "Cholera" on Contact Directory Page
+    And I click APPLY BUTTON in Contact Directory Page
+    And I check that number of displayed contact results is 0
+    And I apply Disease of source filter "COVID-19" on Contact Directory Page
+    And I apply Disease variant filter to "B.1.617.3" on Contact Directory Page
+    And I click APPLY BUTTON in Contact Directory Page
+    And I check that number of displayed contact results is 0
+    And I apply Disease variant filter to "Sequenzierung ausstehend" on Contact Directory Page
+    And I apply Classification of source case filter to "D. Labordiagnostisch bei nicht erfüllter Klinik" on Contact Directory Page
+    And I click APPLY BUTTON in Contact Directory Page
+    And I check that number of displayed contact results is 0
+    And I apply Classification of source case filter to "0. Nicht klassifiziert" on Contact Directory Page
+    And I apply Follow-up status filter to "Keine Nachverfolgung" on Contact Directory Page
+    And I click APPLY BUTTON in Contact Directory Page
+    And I check that number of displayed contact results is 0
+    And I apply Follow-up status filter to "Nachverfolgung abgebrochen" on Contact Directory Page
+
+  @issue=SORQA-5911 @env_de
+  Scenario: Check all filters are working properly in Contacts directory
+    Given I log in with National User
+    When I click on the Contacts button from navbar
+    And I click on the NEW CONTACT button
+    And I fill a new contact form for DE version
+    And I click SAVE button on Edit Contact Page
+    Then I check the created data is correctly displayed on Edit Contact page for DE version
+    And I click on BESTÄTIGTER KONTAKT radio button Contact Person tab
```

Comment:
It it generally not advised to use a different language in the step definition. It's better to write something like "I click on the German button for Confirmed Contact", so that someone who doesn't speak German can make sense of what the scenario is doing. Same applies to all the other mixed-language steps below.

Reply:
True, but in this case we are sharing this method across instances cause it enables clicking web element by text (BESTÄTIGTER KONTAKT is a parameter that you can define in this test step)

Reply:
agree with @jp-sgent , is not ok to use any other language in the BDD layer excepting english, because when we'll check the steps, we won't be able to understand what is there.
Please use the main one, and adapt the locator so it won't be taken by ui text.